eopdev/ap-southeast-2/eopdev/data-stores/kafka/terragrunt.hcl

```diff
@@ -0,0 +1,21 @@
+# Include the root `terragrunt.hcl` configuration, which has settings common across all environments & components.
+include "root" {
+  path = find_in_parent_folders()
+}
+
+# Include the component configuration, which has settings that are common for the component across all environments
+include "envcommon" {
+  path = "${dirname(find_in_parent_folders())}/_envcommon/data-stores/kafka.hcl"
+  # We want to reference the variables from the included config in this configuration, so we expose it.
+  expose = true
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Module parameters to pass in. Note that these parameters are environment specific.
+# ---------------------------------------------------------------------------------------------------------------------
+inputs = {
+  name = "kafka-eopdev"
+  client_sasl_scram_secret_arns = [
+    "arn:aws:secretsmanager:ap-southeast-2:657968434173:secret:AmazonMSK_EOPIngestAPIKafkaCredentials-kndW0a"
+  ]
```

Comment:
This secret was provisioned manually

Reply:
Sweet, some of the security review stuff we've got from setting up the NZISM review pings us for ever using the default KMS key. 
Seems like good standard is never use the default, but this is what default gruntworks was